datasets/li/posting_sanctions/li_entsg.yml

```diff
@@ -32,12 +29,11 @@ publisher:
   country: li
   url: https://www.llv.li/de/landesverwaltung/amt-fuer-volkswirtschaft
   official: true
-
 url: https://www.llv.li/de/unternehmen/berichtspflichten-bewilligungen/grenzueberschreitende-dienstleistungen/entsendung-von-arbeitnehmern/sanktionsliste-gemaess-art.-7-abs.-4-entsg
```

Comment:
this seems to be 404. now https://www.llv.li/de/landesverwaltung/amt-fuer-volkswirtschaft/grenzueberschreitende-dienstleistungen/entsendung-von-arbeitnehmern/sanktionsliste-gemaess-art.-7-abs.-4-entsg